lighthouse-extension/app/src/lighthouse-ext-background.js

```diff
@@ -5,46 +5,20 @@
  */
 'use strict';
 
-require('./lighthouse-background');
+const background = require('./lighthouse-background');
 
 const ExtensionProtocol = require('../../../lighthouse-core/gather/connections/extension');
 const log = require('lighthouse-logger');
 const assetSaver = require('../../../lighthouse-core/lib/asset-saver.js');
 
+/** @typedef {import('../../../lighthouse-core/gather/connections/connection.js')} Connection */
+
 const STORAGE_KEY = 'lighthouse_audits';
 const SETTINGS_KEY = 'lighthouse_settings';
 
-// let installedExtensions = [];
-let disableExtensionsDuringRun = false;
 let lighthouseIsRunning = false;
-let latestStatusLog = [];
-
-// /**
```

Comment:
this was disabled in February 2017. I don't think we need to leave it in anymore just in case :)

Reply:
HERO